components/clarity-lsp/src/common/state.rs

```diff
@@ -240,6 +317,30 @@ impl EditorState {
 
         (contracts, tldr)
     }
+
+    pub fn insert_active_contract(
+        &mut self,
+        contract_location: FileLocation,
+        clarity_version: ClarityVersion,
+        source: &str,
+    ) {
+        let epoch = StacksEpochId::Epoch21;
```

Comment:
The LSP already enforces Epoch21 to compute diagnostic, it should be safe to enforce it here as well